utils/testing.ts

```diff
@@ -36,9 +36,8 @@ export async function compareFormData(formData: Record<string, unknown>, entries
     // Read the file asynchronously.
     const reader = new FileReader();
     reader.onload = function (e) {
-      if (e.target === null) {
-        throw new Error("e.target === null");
-      }
+      expect(e.target).not.toBeNull();
+      if (e.target === null) return; // Make the null-check happy.
```

Comment:
I assume typescript complains on `e.target.result` that `e.target` can be null? I would just drop this `if` and replace that expect below with `expect(e.target?.result).toEqual(expectedData);`

```javascript
> e = {}
{}
> e.target.result
Uncaught TypeError: Cannot read property 'result' of undefined
> e.target?.result
undefined
```